src/Gridly/gridzone.scss

```diff
@@ -0,0 +1,92 @@
+@import '../globals/breakpoints';
+@import '../globals/variables';
+
+
+@mixin respond-to-xs {
+  @media (width <= 576px) {
+    @content;
+  }
+}
```

Comment:
que sea un funcion que acepte un parametro para que uses un solo respon-to revisa el codigo _mixis.scss